src/ngraph/ngraph_sgcompiler.cc

```diff
@@ -197,7 +197,16 @@ std::shared_ptr<ngraph::Function> SGCompiler::MakeForwardFunction(
   }
 
   // create the Forward Function object representing the graph
-  return std::make_shared<ngraph::Function>(outputs, parameters);
+  auto func = std::make_shared<ngraph::Function>(outputs, parameters);
+
+  //fuse conv + bias before autodiff
+  if (sub_graph->context_ == mxnet::Context::CPU()) {
+    ngraph::pass::Manager pass_manager;
+    pass_manager.register_pass<ngraph::runtime::cpu::pass::CPUFusion>(runtime::cpu::pass::CPUFusion::DIFFERENTIABLE_FUSIONS);
```

Comment:
you're missing an `ngraph::` namespace on the enum

Reply:
dang... i only tested it with ngraph tests 😄  and obviously it works 😄 there

Reply:
okay, fixed! This is PR is to mostly serve as a reminder for you that we would like to have some infrastructure for the cases such as conv + bias  more than anything else